### ISSUE TITLE
Capturing Tapes via DVRescue (GUI) on systems other than MacOS

### ISSUE THREAD

**Issue by Rysiek18** [mipops/dvrescue#725]
Hi,

At the outset, I would like to say that I am impressed by the scale and technical advancement of DVRescue. I have been looking for a program similar to yours for a long time, and when I found out about its existence, I thought all my problems with MiniDV tapes had been solved.

Of course, we are talking about overwriting the wrong parts of the recordings without the need to rip the whole tape again. This feature seems particularly useful as my archive consists of about 20 MiniDV tapes and almost all of them are recorded in LP mode. It also doesn't help that the oldest tapes are 20 years old and my Canon MV590 (PAL) camcorder is working very hard (frequent clogging of the head).

Getting to the point: I would like to rip my tapes using your program, but the Capture mode on Windows 10 version 22.12 (GUI) does not work at all, and when downloading the Daily Patches (GUI), my camera is not detected.

I read on your forum that this function works on Mac OS. I specifically purchased an old used MacBook 2.1 running natively on OS X 10.7.5 and tried to update it to the latest possible OS version (El Capitan) but after a few days of trying I had to give up.
Moreover dealing with MacOS itself can be very irritating for ex. when you want to simply transfer files to any other device that is not in Apple ecosystem.

Of course I can send the tapes for professional dubbing, but the cost of dubbing them in the original DV container with all the metadata will be horrendous + the company will probably rip the tapes "as is”, because they don't care about these tapes like I do.

To sum up:
• Do you know when it will be possible to rip tapes with ordinary camcorders (not expensive professional recorders) using DVRescue for Win 10 with the function of overwriting the wrong parts of recordings without the need to rip the whole tape?
(Or is there a Daily Patch version that supports this?)
• Do you have/recommend any other solution for my case?

Regards,
Krzysiek

**Comment by mike86325** [comment]
Hi Krzysiek, 

I'm not a dev, I'm actually just another user that is in the exact same boat as you. You can read my story under issue #718  . I tried Windows 10, 7, and Linux (ubuntu) with different builds and none of them ever worked. I know that they state all these operating systems are supported, but they are really more in an alpha state it seems. 

I also thought of buying my own Mac since the build for MacOS appears to be most functional. I'm sorry you went down this rabbit hole and didn't get it it to work. Did DVRescue not work on the old OS X 10.7.5? Did you ever get the Macbook updated to something newer? I tried to ask in my own issue thread what is the correct combination of Mac hardware, OS, and software build that is going to be functional, but never got a reply. I don't think there's anything on the PC side of things that can be done right now and I'm not sure how close they are to figuring it out. I've tried the nightly builds with no luck as well. 

I gave up and transferred all of my tapes using Scenalyzer 4, but I still have artifacts on a lot of scenes and would really like to recapture everything again using this software. If you end up figuring out how to make it work, please let me know. I think the only option right now is a newer Mac, but then again that is only a guess based on what I think they are using for developing/testing.

-Mike

**Comment by Rysiek18** [comment]
Hi Mike,

Thanks for your answer. I really did not try using it on 10.7.5, cause the oldest officially supported version of Mac OS is 10.12 (And while trying to get some basic features working, before trying to update to 10.11, I couldn't install official web browser, so I supposed that DVRescue won't work)

Now I have blank old Macbook, without any OS (Of course I can reinstall 10.7.5, but what's the point?).
Another thing is that even if I got a newer Macbook with supported OS, got needed adapters and installed DVRescue the question is if my camcorder would even be recognized by it? I don't have hundreds of dollars to spend, just to realize it doesn't work.

This software has enormous potential in DV "home-industry", but they should have prioritized more open OSs with general camcorder support (not professional [and very expensive] decks).

Krzysiek

**Comment by JohnstonJ** [comment]
Here's how I've gotten started using DVRescue on Windows: https://github.com/mipops/dvrescue/issues/897#issuecomment-2264765485

TL;DR: use WinDV to do the actual capture into a single, large AVI file.  Then, use FFmpeg to extract the raw DV data from the AVI file into a `.dv` file that is readily ingested by DVRescue.

**Comment by libbyhopfauf** [comment]
> To sum up: • Do you know when it will be possible to rip tapes with ordinary camcorders (not expensive professional recorders) using DVRescue for Win 10 with the function of overwriting the wrong parts of recordings without the need to rip the whole tape? (Or is there a Daily Patch version that supports this?) • Do you have/recommend any other solution for my case?
> 

@Rysiek18 unfortunately, the Windows side of things will require additional research and development. It is possible to use Ubuntu in a duel boot on a Windows computer for capture. The Windows/Linux version [requires different background tools for that OS](https://docs.google.com/document/d/1zTqIBF-FyKaynIBgYfpBiRmOq5rkbv0S89Uy2p5YsrQ/edit#heading=h.qzakjnqnavln) and you might need to check the [Ubuntu permissions](https://github.com/mipops/dvrescue/issues/514). However, so far this has only consistently resolved capture on Windows for the CLI [as noted in our documentation](https://mipops.github.io/dvrescue/sections/capture.html). Please let us know if this allows you to capture via Windows with a duel boot using the DVRescue GUI. Testing would be helpful.
